packages/@icon-magic/logger/src/index.ts

```diff
@@ -0,0 +1,47 @@
+import { createLogger, format, transports } from 'winston';
+import * as winston from 'winston';
+
+const { combine, colorize, printf, timestamp } = format;
+
+const myFormat = printf(({ level, message, label, timestamp }) => {
+  return `${timestamp} [${label}] ${level}: ${message}`;
+});
+
+/**
+ * Initializing winston's logger to capture the run and error logs
+ */
+export const winstonLogger: winston.Logger = createLogger({
+  transports: [
+    new transports.Console({ level: 'debug' }),
+    new transports.File({ filename: 'icon-magic-run.log', level: 'silly' }),
+    new transports.File({ filename: 'icon-magic-error.log', level: 'error' })
+  ],
+  format: combine(colorize(), timestamp(), myFormat)
+});
+
+/**
+ * Wrapper around winston's logger
+ */
+export interface Logger {
+  error: Function;
```

Comment:
These logger methods could be more specifically defined...
`error: (msg: string) => void`